### DISCUSSION TITLE
Can't login to LGTM with external account after removal of password login!

### DISCUSSION THREAD

**Discussion by ariccio** [community/community#23729] [New to GitHub]
<p>LGTM apparently removed the option to login with an email/password combo several months ago. In the time since they did that, I haven’t used LGTM, so I haven’t been able to connect an external GItHub or Google account.</p>
<p>…but I can’t login with an external account, since my email address is already in use from the password login days, and I can’t connect an external account because I can’t login with my old password. This is a catch-22 bug.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/1/d1cf4fdc7addd108a2a642b25f13b1ba38892e7e.png" data-download-href="/uploads/short-url/tW3XdzTF3cujOftx0oDrADIbZmK.png?dl=1" title="Screenshot 2020-10-13 143739" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2142308/181078931-6d6b8fa4-27c0-4d3e-9ca4-a19eb432e00b.png" alt="Screenshot 2020-10-13 143739" data-base62-sha1="tW3XdzTF3cujOftx0oDrADIbZmK" width="690" height="373" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/1/d1cf4fdc7addd108a2a642b25f13b1ba38892e7e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-10-13 143739</span><span class="informations">752×407 13.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>How can I proceed?</p>

**Comment by chrisgavin** [comment, marked as answer]
<p>Hi <a class="mention" href="/ariccio">@ariccio</a>,</p>
<p>Sorry you are having an issue logging in. Near the end of 2019 we removed support for password-based logins from LGTM to increase security of accounts by leveraging existing support for two-factor authentication and suspicious login detection provided by external authentication providers such as GitHub and Google.</p>
<p>We’ve connected your LGTM account to the GitHub account with a corresponding verified email address, so you should now be able to use the “Login with <a href="http://github.com/" rel="noopener nofollow ugc">GitHub.com</a>” option to regain access to your old account and then link or unlink any other external accounts you would like to.</p>
<p>Apologies again for the inconvenience. Please let us know if you have any other issues logging in or there’s anything else we can help you with.</p>
<p>Kind regards,<br>
Chris</p>
